variables.tf

```diff
@@ -1 +1,248 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
 
+variable "name" {
+  type        = string
+  description = "Name of the vault stack, will be use to prefix resources"
+}
+
+variable "environment" {
+  type        = string
+  description = "Type of environment -- must be one of: dev, test, prod"
+}
+
+variable "key_pair_name" {
+  type        = string
+  description = "Keypair to associate to launched instances"
+}
+
+variable "ami_owner" {
+  type        = string
+  description = "Account id/alias of the AMI owner"
+}
+
+variable "additional_ips_allow_inbound" {
+  type        = list(string)
+  description = "List of ip address that allow to have access to resources"
+  default     = []
+}
+
+variable "ec2_extra_security_group_ids" {
```

Comment:
not used?

Reply:
yes, currently using it, in case users want to add any extra security group to the instances.

Reply:
When I search the PR, I see only this one occurrence of the term. Where exactly is this? Same for others I commented on...

Reply:
Argh, the main.tf was collapsed